src/HTTP/FileClient.php

```diff
@@ -70,7 +70,7 @@ public function request(string $method, string $url, array $headers = []): Respo
         }
 
         if (!$file->success) {
-            throw new HttpException($file->error);
+            throw new HttpException($file->error, $file->get_status_code());
```

Comment:
```suggestion
            throw new HttpException($file->error, $file->get_status_code());	// FreshRSS https://github.com/simplepie/simplepie/pull/905
```

To be removed when changed upstream. Help me keeping track of changes